src/visitor/aa_results.cpp

```diff
@@ -8,15 +8,29 @@
 
 namespace tinytc {
 
-aa_results::aa_results(std::unordered_map<value_node *, value_node *> alias)
-    : alias_(std::move(alias)) {}
+aa_results::aa_results(std::unordered_map<value_node *, value_node *> alias,
+                       std::unordered_map<value_node *, allocation> allocs)
+    : alias_(std::move(alias)), allocs_(std::move(allocs)) {}
 value_node *aa_results::root(value_node &a) {
     auto root = &a;
     if (alias_.find(root) != alias_.end()) {
         root = alias_[root];
     }
     return root;
 }
-bool aa_results::alias(value_node &a, value_node &b) { return root(a) == root(b); }
+bool aa_results::alias(value_node &a, value_node &b) {
```

Comment:
I guess expected signature is non-const for this binary function? Or perhaps `value_node` is const-qualified already.

Reply:
Changed to const